lib/Reconciler.lua

```diff
@@ -539,4 +541,4 @@ function Reconciler._setRbxProp(rbx, key, value, element)
 	end
 end
 
-return Reconciler
\ No newline at end of file
+return Reconciler
```

Comment:
It looks like files' trailing newlines keep getting shuffled there and back, I feel like I should normalize those.

In the meantime, do you have an EditorConfig plugin enabled for your editor?

Reply:
I do, but sometimes it doesn't work (see all the times my commits sometimes get spaces instead of tabs).

Reply:
What editor do you use? The one for Sublime Text seems to horrifically break the automatic indentation detection. :(

Reply:
VS Code and [this](https://github.com/editorconfig/editorconfig-vscode) extension.